driver/razerkbd_driver.c

```diff
@@ -552,6 +552,67 @@ static ssize_t razer_attr_read_game_led_state(struct device *dev, struct device_
     return sprintf(buf, "%d\n", response.arguments[2]);
 }
 
+/**
+ * Write device file "keyswitch_optimization"
+ */
+static ssize_t razer_attr_write_keyswitch_optimization(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    struct usb_interface *intf = to_usb_interface(dev->parent);
+    struct usb_device *usb_dev = interface_to_usbdev(intf);
+    struct razer_report report = {0};
+    unsigned char mode = (unsigned char)simple_strtoul(buf, NULL, 10);
+
+    // Toggle Keyswitch Optimization
+    switch(usb_dev->descriptor.idProduct) {
+    case USB_DEVICE_ID_RAZER_HUNTSMAN_V2:
+        report = razer_chroma_misc_set_keyswitch_optimization_command1(mode);
+        report.transaction_id.id = 0x1f;
+        razer_send_payload(usb_dev, &report);
+        report = razer_chroma_misc_set_keyswitch_optimization_command2(mode);
+        report.transaction_id.id = 0x1f;
+        razer_send_payload(usb_dev, &report);
+        break;
+    default:
+        report = razer_chroma_misc_set_keyswitch_optimization_command1(mode);
+        report.transaction_id.id = 0x1f;
+        razer_send_payload(usb_dev, &report);
+        report = razer_chroma_misc_set_keyswitch_optimization_command2(mode);
+        report.transaction_id.id = 0x1f;
+        razer_send_payload(usb_dev, &report);
```

Comment:
Right now the default doesn't make any sense because it's the same as huntsman v2 case. Did you mean to not set the transaction id here maybe?

Reply:
I went ahead and removed that default if that's ok. I could also put it back and simply remove the transaction IDs. But there aren't any known devices that use those commands without the transaction ID of ox1f.

Reply:
I've added this, I think it's a good idea
```c
    default:
        return -ENOSYS;
```